### PR TITLE
3.8.0 changelog and version

### PR DESCRIPTION
**Pull request by Bob-Murphy**
Updates changelog and version for 3.8.0 update

Also added 3.7.1 changelog here as it was only direcly added to Master branch in the past

### CODE REVIEW

File: changelog.rst [hunk 1]
```diff
@@ -3,6 +3,99 @@ Changelog
...
 
+Version 3.8.0
+===============
+
+.. note::
+   xxth December 2024
+
+.. admonition:: Major
+
+   * :issue:`3441` & :issue:`3442` Added Expeditionary Forces MJTF factions plus Vanilla and Western Sahara compatibility
```
Comment (ante185) [inline changelog.rst:14]: clarify/reword that it's EF intigration into the vanilla and WS factions

File: changelog.rst [hunk 2]
```diff
@@ -3,6 +3,99 @@ Changelog
...
+
+   * :issue:`3376` Changed to have more frequent militia vehicles/transports early game
+   * :issue:`3388` Implemented mortar/artillery support ranging shot and walking barrage
+   * :issue:`3389` Added injector success and UID logging
+   * :issue:`3430` Turned Carpet bombing into triple airstrike
+ 
+.. admonition:: Template updates
+
+   * :issue:`3182` Added 3CB CSAT Scimitar, Gryphon, Bear, and Viper regiment
+   * :issue:`3407` Updated Vanilla/WS Factions, fine tuning, officer loadouts, further RF integration on missed factions
```
Comment (ante185) [inline changelog.rst:27]:
```suggestion
   * :issue:`3407` Updated Vanilla/WS Factions, fine tuning, officer loadouts, further RF integration in vanilla and WS factions
```

**Review by ante185 — CHANGES_REQUESTED** [review]

File: changelog.rst [hunk 3]
```diff
@@ -3,6 +3,99 @@ Changelog
...
+   * :issue:`3390` Fixed HQ actions being available through remote controls
+   * :issue:`3393` Fixed skill upgrade tooltip not updating
+   * :issue:`3395` Fixed errors caused by tags function
+   * :issue:`3397` Fixed arsenal not clearing client IDs on disconnect
+   * :issue:`3402` Prevented rebels AIs being equipped with VN melee weapons
+   * :issue:`3406` Fixed base class for srifle_GM6_snake_lxWS
+   * :issue:`3418` Fixed precedence error in mortar/arty support routine
+   * :issue:`3416` Fixed custom WS Kamaz Textures
+   * :issue:`3436` Fixed debug line in fn_setPlaneLoadout.sqf
+   * :issue:`3443` Fixed EF CDLC detection
```
Comment (Tiny-DM) [inline changelog.rst:52]: This PR should be dropped. It was a hotfix for EF support

File: changelog.rst [hunk 4]
```diff
@@ -3,6 +3,99 @@ Changelog
...
+   * :issue:`3414` Added Aegis Factions Compatibility
+   * :issue:`3428` Added SPEX Factions Templates
+   * :issue:`3442` Added 3CBF weapon integration into RHS factions
+   * :issue:`3437` Updated IFA Faction Templates
+   * :issue:`3451` Fixed CUP template in BAF, ION
+   * :issue:`3452` Updated 3CB Ranger and fixed artillery
+
+.. admonition:: Terrain updates
+
+   * :issue:`3424` & :issue:`3448` Updated Isla Duala to hide "bad" objects and cull obsolete enemy locations
```
Comment (ante185) [inline changelog.rst:38]: This also enables/unhides the Isla Duala mission

File: changelog.rst (hunk 1 above)
Comment (Bob-Murphy) [reply changelog.rst]: done

File: changelog.rst (hunk 2 above)
Comment (Bob-Murphy) [reply changelog.rst]: done

File: changelog.rst (hunk 3 above)
Comment (Bob-Murphy) [reply changelog.rst]: done

File: changelog.rst (hunk 4 above)
Comment (Bob-Murphy) [reply changelog.rst]: done

File: changelog.rst [hunk 5]
```diff
@@ -3,6 +3,110 @@ Changelog
...
 
+Version 3.8.0
+===============
+
+.. note::
+   xxth December 2024
+
+.. admonition:: Major
+
+   * :issue:`3441` & :issue:`3442` Added Expeditionary Forces MJTF factions and added EF integration Vanilla and Western Sahara templates
```
Comment (ante185) [inline changelog.rst:14]:
```suggestion
   * :issue:`3441` & :issue:`3442` Added Expeditionary Forces MJTF factions and added EF integration in Vanilla and Western Sahara templates
```

File: changelog.rst [hunk 6]
```diff
@@ -3,6 +3,110 @@ Changelog
...
+   * :issue:`3392` Added UI and supporting code to set custom loadouts for rebel troops
+
+.. admonition:: Minor
+
+   * :issue:`3376` Changed to have more frequent militia vehicles/transports early game
```
Comment (ante185) [inline changelog.rst:19]: this effect war level 1-4, worthwhile to note maybe?

File: changelog.rst [hunk 7]
```diff
@@ -3,6 +3,110 @@ Changelog
...
+
+.. admonition:: Template updates
+
+   * :issue:`3182` Added 3CB CSAT Scimitar, Gryphon, Bear, and Viper regiment
+   * :issue:`3407` Updated Vanilla/WS Factions, fine tuning, officer loadouts, further RF integration in vanilla and WS factions
+   * :issue:`3413` & :issue:`3429` & :issue:`3460` Added CWR3 Factions Templates
+   * :issue:`3414` Added Aegis Factions Compatibility
+   * :issue:`3428` Added SPEX Factions Templates
+   * :issue:`3442` Added 3CBF weapon integration into RHS factions
+   * :issue:`3437` Updated IFA Faction Templates
```
Comment (ante185) [inline changelog.rst:34]:
```suggestion
   * :issue:`3437` Updated IFA Faction Templates to solve a bug with the large intel item. Adds Churchills to the IFA UK faction IFV lineup
```

File: changelog.rst [hunk 8]
```diff
@@ -3,6 +3,110 @@ Changelog
...
+   * :issue:`3451` Fixed CUP template in BAF, ION
```
Comment (ante185) [inline changelog.rst:35]:
```suggestion
   * :issue:`3451` Fixed missing comma and incorrect artillery classnames in CUP template in BAF, ION
```

File: changelog.rst [hunk 9]
```diff
@@ -3,6 +3,110 @@ Changelog
...
+   * :issue:`3452` Updated 3CB Ranger and fixed artillery
```
Comment (ante185) [inline changelog.rst:36]:
```suggestion
   * :issue:`3452` Updated 3CB LDF with the Livonian Armed Forest Rangers, and corrected magazine classname for the rocket artillery
```

File: changelog.rst [hunk 10]
```diff
@@ -3,6 +3,110 @@ Changelog
...
+   * :issue:`3457` Fixed NVG entry in RHS HIDF and flagged RHS HIDF and TLA as lowtech
```
Comment (ante185) [inline changelog.rst:37]:
```suggestion
   * :issue:`3457` Fixed NVG entry in RHS HIDF
```
My bad, did not keep the PR description up to date

**Review by ante185 — COMMENTED** [review]
Some things i appear to have glossed over when i last looked through these

File: changelog.rst (hunk 6 above)
Comment (Tiny-DM) [reply changelog.rst:19]: eh, its implied by militia imo

File: changelog.rst [hunk 11]
```diff
@@ -3,6 +3,110 @@ Changelog
...
+   * :issue:`3441` & :issue:`3442` Added Expeditionary Forces MJTF factions and added EF integration Vanilla and Western Sahara templates
+   * :issue:`3392` Added UI and supporting code to set custom loadouts for rebel troops
+
+.. admonition:: Minor
+
+   * :issue:`3376` Changed to have more frequent militia vehicles/transports early game
+   * :issue:`3388` Implemented mortar/artillery support ranging shot and walking barrage
+   * :issue:`3389` Added injector success and UID logging
+   * :issue:`3430` Turned Carpet bombing into triple airstrike
+   * :issue:`3459` Added various TFAR bagpacks and to multiple templates
```
Comment (Tiny-DM) [inline changelog.rst:23]: backpacks? bagpacks?

File: changelog.rst (hunk 6 above)
Comment (Bob-Murphy) [reply changelog.rst:19]: good as is as it's mentioning militia as well as early game

**Review by ante185 — APPROVED** [review]
lgtm i think

**Pull request merged**